src/style.css

```diff
@@ -1,3 +1,64 @@
 body {
   margin: 0;
 }
+
+header {
+  padding: 0 50px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav_list {
```

Comment:
Avoid using tag names for styling (except `html` and `body`). Instead, style elements using `.class`. Replace `.nav_list` with a class that doesn't resemble a tag name.